packages/core/build/package.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
+}
```

Comment:
This is used for ts-node which will allow our build scripts to use TS instead of JS if desired. TS helped a lot with the tokens to ensure type checking the references between aliases.